sdk/core/azure-core/test/libcurl-stress-test/README.md

```diff
@@ -15,9 +15,9 @@ The helm chart creates a pod with a container based on the docker image, and exe
 
 To deploy the chart you will need to run "azure-sdk-for-cpp\eng\common\scripts\stress-testing> .\deploy-stress-tests.ps1 -Namespace azuresdkforcpp -SearchDirectory E:\src\azure-sdk-for-cpp\sdk\core\azure-core\test -PushImage"
 
-Where namaspace will be created if missing , search directory can be any folder where it will search for charts in it and all it's sub dirs, push image will call it to build the docker image. 
+Where name space will be created if missing , search directory can be any folder where it will search for charts in it and all it's sub dirs, push image will call it to build the docker image. 
 
-ATM the docker image is build by hand and harcoded in the chart to simplify matters.  
```

Comment:
I would prefer either "hardcoded" or "hard-coded". The original complaint was about "harcoded".